modules/docs/arrow-docs/docs/docs/arrow-core/typeclasses/monadplus/README.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: docs-core
+title: MonadPlus
+permalink: /docs/arrow/typeclasses/monadplus/
+redirect_from:
+  - /docs/typeclasses/monadplus/
+---
+
+## MonadPlus
+
+The `MonadPlus` typeclass lets us combine two structures of type `Kind<F, A>` with a associative function `plusM`
+into a `Kind<F, A>`. It also provides the function `zeroM` which acts as a neutral argument to the `plusM` function. 
```

Comment:
Some comment on the relation between this and Alternative would be good. Just note how this is basically Alternative with a Monad constraint and additional laws.